### ISSUE TITLE
iekt代码部分，部分数据在cpu上，部分在cuda上

### ISSUE THREAD

**Issue by 123kristin** [pykt-team/pykt-toolkit#129]
代码修改：y,loss = model.train_one_step(data, device)后续一路添加

**Comment by Yang-HangWA** [comment]
遇到同样的错误，貌似加了device参数还是会报错。

**Comment by wei45955** [comment]
改成这样就好了，也可以改成.to(device)
```
    def batch_to_device(self,data,process=True):
        if not process:
            return data
        dcur = data
        # q, c, r, t = dcur["qseqs"], dcur["cseqs"], dcur["rseqs"], dcur["tseqs"]
        # qshft, cshft, rshft, tshft = dcur["shft_qseqs"], dcur["shft_cseqs"], dcur["shft_rseqs"], dcur["shft_tseqs"]
        # m, sm = dcur["masks"], dcur["smasks"]
        data_new = {}
        data_new['cq'] = torch.cat((dcur["qseqs"][:,0:1], dcur["shft_qseqs"]), dim=1).cuda()
        data_new['cc'] = torch.cat((dcur["cseqs"][:,0:1],  dcur["shft_cseqs"]), dim=1).cuda()
        data_new['cr'] = torch.cat((dcur["rseqs"][:,0:1], dcur["shft_rseqs"]), dim=1).cuda()
        data_new['ct'] = torch.cat((dcur["tseqs"][:,0:1], dcur["shft_tseqs"]), dim=1).cuda()
        data_new['q'] = dcur["qseqs"].cuda()
        data_new['c'] = dcur["cseqs"].cuda()
        data_new['r'] = dcur["rseqs"].cuda()
        data_new['t'] = dcur["tseqs"].cuda()
        data_new['qshft'] = dcur["shft_qseqs"].cuda()
        data_new['cshft'] = dcur["shft_cseqs"].cuda()
        data_new['rshft'] = dcur["shft_rseqs"].cuda()
        data_new['tshft'] = dcur["shft_tseqs"].cuda()
        data_new['m'] = dcur["masks"].cuda()
        data_new['sm'] = dcur["smasks"].cuda()
        return data_new
```
